app/src/main/java/org/wikipedia/dataclient/SharedPreferenceCookieManager.kt

```diff
@@ -105,6 +106,10 @@ class SharedPreferenceCookieManager(
                 buildCookieList(cookieList, cookiesForDomainSpec, CENTRALAUTH_PREFIX)
             }
         }
+        if (LoginClient.enqueueForceEmailAuth && (url.toString().contains("action=clientlogin"))) {
```

Comment:
This is some not-very-clean logic for injecting a cookie for "forcing" email authentication. Unfortunately this is basically the only way it can be done.